lametro/events.py

```diff
@@ -277,8 +275,12 @@ def scrape(self, window=None) :
                                media_type="application/pdf")
 
             if event['EventMinutesFile']:
-                e.add_document(note= 'Minutes',
-                               url = event['EventMinutesFile'],
+                e.add_document(note='Minutes',
+                               url=event['EventMinutesFile'],
+                               media_type="application/pdf")
+            elif web_event['Published minutes'] != 'Not\xa0available':
```

Comment:
Are EventMinutesFile and Published minutes mutually exclusive?

Reply:
i don't know if they are, but it seems like they **should** be.

Reply:
Ha, that's a good point. Still, the previous logic adds them both, and we haven't seen duplicates, AFAICT. What do you think about preserving that functionality?

Reply:
doesn't the fact that we haven't seen duplicates indicate that they are mutually exclusive?

I think this it the logic that we want. If the API has the EventsMinuteFile, then those should be the minutes.

Reply:
i guess the question is, what do we want the behavior to be if they both exist. 

Two scenarios:

1. they have different content.  (then which ones are the minutes?
2. they link the same content, but tthe url would be not be same.
3. the urls are the same (this case seems easy)

Reply:
For context, I'm kind of relying on `notes="Minutes"` in my refactor, but we don't need that for current la metro. so we can also put on this now.

Reply:
i think punting for now and asking for metro's clarification/input on behavior would be ideal!

Reply:
sound good, done.